test/reporter-html.js

```diff
@@ -106,7 +106,7 @@ QUnit.test( "setup", function( assert ) {
 QUnit.test( "basics", function( assert ) {
 	assert.expect( 2 );
 	var previous = getPreviousTests( /^setup$/, /^timing$/ )[ 0 ],
-		runtime = previous.lastChild.previousSibling;
+		runtime = previous.lastChild.previousSibling.previousSibling;
```

Comment:
Here's a good opportunity to include tests for these changes.

Would you include a test with a regexp containing the static part of the source line?